src/pam_hbac.h

```diff
@@ -93,5 +94,7 @@ void free_string_list(char **list);
 size_t null_string_array_size(char *arr[]);
 size_t null_cstring_array_size(const char *arr[]);
 void logger(pam_handle_t *pamh, int level, const char *fmt, ...);
+void logger2(pam_handle_t *pamh, int level, const char *fmt, va_list ap);
```

Comment:
Surely we can have a better function name :) Even va_logger() or vlogger() seem better to me than logger2..